spec/page_element_spec.rb

```diff
@@ -22,20 +22,41 @@
       )).pages.first
   end
 
-  context '#title' do
-    context 'no metadata' do
+  let(:page2) do
+    book_containing(html:
+      <<~HTML
+        <div data-type="page">
+          <div data-type="document-title">Title</div>
+          <div data-type="metadata">
+            <div data-type="document-title">Title MetaData</div>
+          </div>
+        </div>
+      HTML
+    )
```

Comment:
I would not name this variable `page2` because it is actually a book.  Leads to `page2.pages[0]` below which looks funny.  

Instead what I would do is name it some more meaningful and drill down to the page here, e.g.

```
let(:page_with_multiple_titles) do 
  book_containing(html:
    <<~HTML
      <div data-type="page">
        <div data-type="document-title">Title</div>
        <div data-type="metadata">
          <div data-type="document-title">Title MetaData</div>
        </div>
      </div>
    HTML
  ).pages.first!
end
```